analytics/tests/test_counts.py

```diff
@@ -1132,19 +1132,20 @@ def test_aggregation(self) -> None:
 
     def test_active_users_log_by_is_bot(self) -> None:
         property = 'active_users_log:is_bot:day'
-        user = do_create_user('email', 'password', self.default_realm, 'full_name')
+        acting_user = self.example_user("iago")
+        user = do_create_user('email', 'password', self.default_realm, 'full_name', acting_user=None)
         self.assertEqual(1, RealmCount.objects.filter(property=property, subgroup=False)
                          .aggregate(Sum('value'))['value__sum'])
-        do_deactivate_user(user)
+        do_deactivate_user(user, acting_user=user)
         self.assertEqual(0, RealmCount.objects.filter(property=property, subgroup=False)
                          .aggregate(Sum('value'))['value__sum'])
-        do_activate_user(user)
+        do_activate_user(user, acting_user=acting_user)
         self.assertEqual(1, RealmCount.objects.filter(property=property, subgroup=False)
                          .aggregate(Sum('value'))['value__sum'])
-        do_deactivate_user(user)
+        do_deactivate_user(user, acting_user=user)
```

Comment:
Why do you randomly switch around which user is acting in these tests?  It makes them very hard to read; I think it'd be cleaner to pass `acting_user=None` in tests unless there's a specific reason not to, since that'd be more readable.

(Also, I think you left off the `,*` part of `def do_deactivate_user`).

Reply:
My thought was that any user (also admin/owner) can deactivate the account but only admin/owner can reactivate it.